### PR TITLE
release v0.7 docs

### PR DESCRIPTION
**Pull request by eahydra**
Signed-off-by: Joseph <joseph.t.lee@outlook.com>

### CODE REVIEW

File: versions.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [
+  "v0.7",
   "v0.6",
   "v0.5",
   "v0.4"
```
Comment (hormes) [inline versions.json:5]: remove v0.4, we keep lastest 3 version here.

Comment (eahydra) [reply versions.json:5]: done

**Pull request merged**